src/components/tikzpictureview.ts

```diff
@@ -0,0 +1,370 @@
+import * as vscode from 'vscode'
+import * as path from 'path'
+import * as fs from 'fs'
+import * as fse from 'fs-extra'
+import * as child_process from 'child_process'
+import { tmpdir } from 'os'
+import { promisify } from 'util'
+
+import { Extension } from '../main'
+
+const mkdtemp = promisify(fs.mkdtemp)
```

Comment:
You do not have to use `promisify` in async functions. Just use `mkdtempSync`.

Reply:
See 78f4527903e16cdba5b6c918b870a4f081f90acd